test/NuGet.Core.Tests/NuGet.Frameworks.Test/NuGet.Frameworks.Test.csproj

```diff
@@ -3,15 +3,11 @@
     <TargetFrameworks>$(TargetFrameworksExe)</TargetFrameworks>
     <UseParallelXunit>true</UseParallelXunit>
     <Description>Unit tests for NuGet.Frameworks.</Description>
-    <UserSecretsId>f889ae50-c1f1-4c63-beae-292414bb3939</UserSecretsId>
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\TestUtilities\Test.Utility\Test.Utility.csproj" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+    <ProjectReference Include="..\..\..\src\NuGet.Core\NuGet.Frameworks\NuGet.Frameworks.csproj" />
+    <ProjectReference Include="..\..\..\src\NuGet.Core\NuGet.Packaging\NuGet.Packaging.csproj" />
```

Comment:
Why do we need NuGet.Packaging?

I love that we're removing the Test.Utility dependency btw: https://github.com/NuGet/Home/issues/13172.

Reply:
This one test uses it?
https://github.com/NuGet/NuGet.Client/pull/5670/files#diff-47c92885734ec4a1cca4fcf64192e06ae010df2b9ffdba2fa9957b11e4f86e45R4

Reply:
The link doesn't work :(

Reply:
😡 
https://github.com/NuGet/NuGet.Client/blob/dev-jeffkl-frameworks-test/test/NuGet.Core.Tests/NuGet.Frameworks.Test/NuGetFrameworkUtilityTests.cs#L5

Reply:
Oh for get nearest since it's parameterized. 

That's ok then even it's not the most perfect technical design ever :D
Thanks for linking that.